### PR TITLE
feature: cards on resources

### PR DESCRIPTION
**Pull request by Paul-Bob**
[#1072](https://github.com/avo-hq/avo/issues/1072)

### CODE REVIEW

**Review by adrianthedev — APPROVED** [review]

File: docs/3.0/resources.md [hunk 1]
```diff
@@ -691,3 +691,34 @@ class Avo::Resources::Project < Avo::BaseResource
...
+    field :id, as: :id
+    field :name, as: :text
+    field :email, as: :text
+    field :roles, as: :boolean_group, options: {admin: "Administrator", manager: "Manager", writer: "Writer"}
+  end
+
+  def cards
+    card Avo::Cards::ExampleAreaChart, cols: 3
+    card Avo::Cards::ExampleMetric, cols: 2
+    return if view.form?
```
Comment (adrianthedev) [inline docs/3.0/resources.md:713]: Let's take this out. If we're trying to keep users from using `if/else` we should not advertise early returns.
Let's use `visible_on` or wtv. we have here.

File: docs/3.0/dashboards.md [hunk 2]
```diff
@@ -55,77 +55,6 @@ The `id` field has to be unique. The `name` is what the user sees in big letters
 
 Using the ' grid_cols ' parameter, you may organize the cards in a grid with `3`, `4`, `5`, or `6` columns using the `grid_cols` parameter. The default is `3`.
```
Comment (adrianthedev) [inline docs/3.0/dashboards.md:57]: Whenever we take out a big chunk of documentation to it's own page, let's leave the heading there (for SEO and findability purposes) with a link to the new section.

File: docs/3.0/cards.md
Comment (adrianthedev) [inline docs/3.0/cards.md:1]: Let's add a reference for `self.format` to this page. We don't have it documented anywhere.

**Review by adrianthedev — CHANGES_REQUESTED** [review]
Left some feedback. We can merge after that.

File: docs/3.0/dashboards.md (hunk 2 above)
Comment (Paul-Bob) [reply docs/3.0/dashboards.md:57]: Added a reference in [this commit](https://github.com/avo-hq/avodocs/pull/110/commits/deb0295885976c8636d55e9a04d5ba813c9696a0)

File: docs/3.0/cards.md
Comment (Paul-Bob) [reply docs/3.0/cards.md:1]: Added a reference in [this commit](https://github.com/avo-hq/avodocs/pull/110/commits/deb0295885976c8636d55e9a04d5ba813c9696a0)

**Pull request merged**